app/src/main/java/de/westnordost/streetcomplete/overlays/cycleway/StreetCyclewayOverlayForm.kt

```diff
@@ -80,13 +88,23 @@ class StreetCyclewayOverlayForm : AStreetSideSelectOverlayForm<CyclewayAndDirect
         }
     }
 
+    private fun onLoadInstanceState(state: Bundle) {
+        bicycleBoulevard = state.getString(BICYCLE_BOULEVARD)
+            ?.let { BicycleBoulevard.valueOf(it) }
+            ?: BicycleBoulevard.NO
+        bicycleInPedestrianStreet = state.getString(BICYCLE_IN_PEDESTRIAN_STREET)
+            ?.let { BicycleInPedestrianStreet.valueOf(it) }
```

Comment:
Why does `bicycleBoulevard` get a fallback value and `bicycleInPedestrianStreet` doesn't?

Reply:
because then it is null. (null means that it is not a pedestrian street, in that case)